ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/useApps.ts

```diff
@@ -45,20 +44,15 @@ export function useApps({ envID, isArchived }: { envID: string; isArchived: bool
           };
         }
 
-        const functionCount =
-          latestSync?.syncedFunctions.filter((fn) => {
-            return !fn.isArchived;
-          }).length || 0;
-
         return {
           ...app,
           latestSync,
-          functionCount,
+          functionCount: app.functions.length,
```

Comment:
Would it be faster to just have `functionsCount` on the resolver and handle this on the backend? The backend would also just be able to run a count query greater than the existing 200 limit 🤔  https://github.com/inngest/monorepo/pull/2162/files#diff-44bdfdd8fa91cc5626a104f0c1031c739dbe4217bcdac2b085d41a728ba4a717R68

Reply:
We'll eventually have an expandable list of functions in the app list page so we'll still need `App.functions`. But since that will eventually be paginated, you're right that we'll also need `App.functionCount`. I'll add that